Sources/Models/SubscribeData.swift

```diff
@@ -3,24 +3,28 @@ import Stripe
 
 public struct SubscribeData: Equatable {
   public var coupon: Stripe.Coupon.Id?
+  public var isOwnerTakingSeat: Bool
   public var pricing: Pricing
   public var teammates: [EmailAddress]
   public var token: Stripe.Token.Id
 
   public init(
     coupon: Stripe.Coupon.Id?,
+    isOwnerTakingSeat: Bool,
     pricing: Pricing,
     teammates: [EmailAddress],
     token: Stripe.Token.Id
   ) {
     self.coupon = coupon
+    self.isOwnerTakingSeat = isOwnerTakingSeat
     self.pricing = pricing
     self.teammates = teammates
     self.token = token
   }
 
-  private enum CodingKeys: String, CodingKey {
+  public enum CodingKeys: String, CodingKey {
```

Comment:
made it public so that I could use these values when building form inputs. helps keep those things in 1-1.